app/Resources/views/base.html.twig

```diff
@@ -2,12 +2,20 @@
 <html>
     <head>
         <meta charset="UTF-8" />
-        <title>{% block title %}Welcome!{% endblock %}</title>
+        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+        <meta content="width=device-width, initial-scale=1.0, user-scalable=no" name="viewport">
+        <meta name="google" content="notranslate" />
+        <meta name="description" content="{% block description %}{{ 'homepage.description' | trans({}, 'seo') }}{% endblock %}">
+        <link type='text/css' href='{{ asset('assets/css/vendor.css') }}' rel='stylesheet' media='screen' />
+        <link type='text/css' href='{{ asset('assets/css/app.css') }}' rel='stylesheet' media='screen' />
+        <title>{% block title %}{{ 'homepage.title' | trans({}, 'seo') }}{% endblock %}</title>
```

Comment:
In the previous project, I noticed that the title block was on the top of all CSS link. I don't know if it is relevant, but I wanted to point it out.
Maybe need a quick check ?
Thank you